pkg/pool-hooks/contracts/hooks-quantamm/HyperSurgeHook.sol

```diff
@@ -686,4 +684,42 @@ contract HyperSurgeHook is BaseHooks, VaultGuard, SingletonAuthentication, Versi
 
         return locals.maxDev;
     }
+
+    /**
+     * @notice Checks if the pool price deviation is worsening after a add/remove liquidity operation.
+     * @dev The pool price deviation is worsening if the deviation between oracle and pool price increased and the
+     * deviation is greater than the surge threshold.
+     * @param pool The pool address
```

Comment:
```suggestion

     * @param pool The pool address
```
Nit - if you're following our NatSpec conventions, there's a blank line after multi-line dev comments.